pkg/cmd/cmdlist/list.go

```diff
@@ -0,0 +1,49 @@
+package cmdlist
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/openshift-eng/openshift-tests-extension/pkg/flags"
+	g "github.com/openshift-eng/openshift-tests-extension/pkg/ginkgo"
+	"github.com/openshift-eng/openshift-tests-extension/pkg/ginkgo/ginkgofilter"
+)
+
+func NewCommand() *cobra.Command {
+	var listOpts struct {
+		all      bool
+		envFlags *flags.EnvironmentFlags
+	}
+	listOpts.envFlags = flags.NewEnvironmentFlags()
```

Comment:
This is an alternative to named, versioned options https://github.com/openshift/enhancements/pull/1676#discussion_r1772050511